src/ophys_etl/modules/suite2p_registration/suite2p_utils.py

```diff
@@ -125,3 +127,209 @@ def compute_reference(frames: np.ndarray,
         )
 
     return ref_image
+
+
+def optimize_motion_parameters(initial_frames: np.ndarray,
+                               smooth_sigmas: np.array,
+                               smooth_sigma_times: np.array,
+                               suite2p_args: dict,
+                               logger: callable = None) -> dict:
+    """Loop over a range of parameters and select the best set from the
+    max acutance of the final, average image.
+
+    Parameters
+    ----------
+    initial_frames : numpy.ndarray, (N, M, K)
+        Smaller subset of frames to create a reference image from.
+    smooth_sigmas : numpy.ndarray, (N,)
+        Array of suite2p smooth sigma values to attempt. Number of iterations
+        will be len(`smooth_sigmas`) * len(`smooth_sigma_times`).
+    smooth_sigma_times : numpy.ndarray, (N,)
```

Comment:
Similarly: do we need to make sure that devs understand that `len(smooth_sigmas)` does not have to be the same as `len(smooth_sigma_times)`?

Reply:
(I get that the last sentence of the two parameter descriptions implies that they are not the same. I'm just not used to specifying the shapes of arrays this way. I'd usually just say "1-dimensional numpy array"in the text)